src/main.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App.tsx";
+import "@styles/index.scss";
+
+ReactDOM.createRoot(document.getElementById("root")!).render(
```

Comment:
what happens when document can't find root id?

Reply:
If document cannot find root id, it returns 'null' and createRoot leads to an error: Target container is not a DOM element. 
But i have a div with id="root" in index.html. Are there any cases that document cannot find root id?

Reply:
Yah I need to handle that case friend, prevent break app